internal/cli/main.go

```diff
@@ -190,8 +191,23 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 			log.fatalErr(migraterErr)
 		}
 
-		limit := -1
-		if flag.Arg(1) != "" {
+		limit := 0
+		if flag.Arg(1) == "" {
+			fmt.Println("Are you sure you want to apply all down migrations? [y/N]")
+
+			reader := bufio.NewReader(os.Stdin)
+			response, err := reader.ReadString('\n')
```

Comment:
Not sure what the difference between this and `fmt.Scanln()` is.

Reply:
switched to `fmt.Scanln`